third_party/blink/renderer/platform/heap_observer_set.h

```diff
@@ -87,6 +99,7 @@ class PLATFORM_EXPORT HeapObserverSet {
   // optionally barring add or remove mutations.
   mutable IterationState iteration_state_ = kNotIterating;
   ObserverSet observers_;
+  HeapHashSet<Member<ObserverType>> replay_strong_observers_;
```

Comment:
Nit: In keeping consistency with other changes, we should probably call this `record_replay_strong_observers_`